common/src/main/java/com/wynntils/features/chat/GuildRankReplacementFeature.java

```diff
@@ -35,7 +35,7 @@ public class GuildRankReplacementFeature extends Feature {
 
     // Test suite: https://regexr.com/7f8kh
```

Comment:
You did not update the test suite..?

Reply:
There should be a negative match for WAR messages.

Also, we should really try to get an internal junit test suite instead of relying on regexr...

Reply:
Also for INFO that case. However, this works because of the color matching so I rather merged it.

Reply:
I'm not saying you did wrong that merged it. I'm just saying we should strive to keep our regex test suites up to date, when we find new cases that cause us to have to change them.

Reply:
I was going to update it but the test that's already there doesn't seem to work and I don't have champion to test how it looks exactly with changed nickname, sorry should've mentioned it